keymanager-client/src/client.rs

```diff
@@ -85,8 +83,9 @@ impl RemoteClient {
 
     /// Create a new key manager client with runtime-internal transport.
     ///
-    /// Using this method automatically obtains valid key manager enclave identities via the
-    /// worker-host protocol.
+    /// Using this method valid enclave identities won't be preset and should
+    /// be obtained via the worker-host protocol and updated with the set_policy
+    /// method.
```

Comment:
Maybe also say that until `set_policy` is called session establishment will fail (I assume this is the case)?